### ISSUE TITLE
[Issue]: 10.8.13 upgraded to 10.9.1 Notifications in admin panel just spins and Network settings are empty

### ISSUE THREAD

**Issue by gummykage** [jellyfin/jellyfin#11628]
### Please describe your bug

After upgrading to 10.9.1 from 10.8.13 the notifications panel in the dashboard / admin panel just spins and doesn't show anything. Network settings is not populated. iOS app you can't scroll from category to category without it freezing for 3-5 seconds. Scan media would continuously fail until i removed a something framework or environment=1 (can't remember the exact syntax was recommended by someone saying it was no longer needed in 10.9.x)

Logs say the network.xml file contains an error.

I rolled everything back to 10.8.13 for now. But hopping this will get some attention. I'm willing to test anything.

Thank you all for your hardwork!

### Reproduction Steps

10.9.1 Docker image on Container Station reading the config/cache files of 10.8.13.
Using Jellyfin Media Player for windows 1.9.1
Using Jellyfin Mobile iOS latest version.

### Jellyfin Version

10.9.0

### Environment

```markdown
Docker pulled 10.9.1 running on Container Station / Qnap TS853-Pro
Client Jellyfin Media Player latest on win10
Client Jellyfin Mobile on iOS latest on 13.5.1
```

### Jellyfin logs

```shell
Rolled it back, but there appears to be incompatibility with the network.xml file as it says it contains an error coming from 10.8.13 to 10.9.1.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Shadowghost** [comment]
You can just remove the network settings xml and restart. When you save it again it will be set to defaults.
The other issue seems to me like a cach issue, clear your browser cache and try again

**Comment by gummykage** [comment]
@Shadowghost The issue with the categories slow to slide happens mostly on ios. I've actually uninstall the app and reinstalled it from scratch and it still does the freezing issue. It's ok with 10.8.13 on my 2nd and 3rd servers.  I will try the network.xml soon. Just restored a backup of 10.8.13 and reverted back to no problems.

**Comment by tgly307** [comment]
https://github.com/jellyfin/jellyfin/issues/11548#issuecomment-2106100017   maybe help

**Comment by gummykage** [comment]
@tgly307 Thank you for the read on that, i wrote all that down because I'm convinced it will happen to me anyways in other instances later, the exact issue I was having was the network.xml file contained some error that 10.9.1 didn't like when updating from 10.8.13. Even though this error would occur it would still bind to my addresses correctly so I had no issue with accessing the server and didn't need to manually change the virtual interfaces to false. Also deleting the network.xml file just creates a default one, and if doesn't you can always get one from a new installation. There is a lot of amazing updates from 10.9.1 that's too hard to pass up, but a ton of new bugs that I'm slowly running into and trying to find workarounds for. For now I went back to 10.8.13 where I can focus watching than pulling my hair out. I have a test docker I'm going through the issues in this github section and trying to reproduce the issue and see if I can find workarounds for people.

**Comment by gummykage** [comment]
Network issue appears to have been solved with 10.9.2. The jellyfin mobile ios client is also no longer freezing.
